build.sbt

```diff
@@ -27,11 +27,24 @@ val macroSettings = Seq(
 )
 
 val sharedSettings = Seq(
-
+  assemblyMergeStrategy in assembly := {
+    case PathList("javax", "servlet", xs@_*) => MergeStrategy.first
+    case m if m.toLowerCase.endsWith("manifest.mf") => MergeStrategy.discard
+    case m if m.toLowerCase.matches("meta-inf.*\\.sf$") => MergeStrategy.discard
+    case _ => MergeStrategy.first
```

Comment:
I'd like to make a guess that the above line is guilty for failures in CI. Also, the previous two line are just the same as in the default merge strategy.  According to the sbt-assembly documentation you should use the default strategy for all cases that don't need special treatment.  Could you please tell what was the original merge conflict you tried to resolve ?